### ISSUE TITLE
Lightning Flash Integration. 

### ISSUE THREAD

**Issue by tchaton** [heartexlabs/label-studio#915]
Dear people from Label-Studio,

First of all, this framework looks really neat ! Congratulations.

I am part of PyTorch Lightning and I wanted to share awareness about Lightning Flash: https://github.com/PyTorchLightning/lightning-flash
Flash is collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning.
All tasks are production ready and will be soon fairly optimized.

There is definitely a great synergy between our frameworks and I think Flash could be a great ML plugin for LabelStudio.
I would be interested to hear your thoughts and possibly collaborate on creating such plugin.

We are also building Grid.ai, https://docs.grid.ai and LabelStudio would be an amazing extension.

By having Grid.ai + LabelStudio + Production Ready Flash Tasks, it would be simpler than ever to bring ML to production for everyone.

Best,
T.C

**Comment by makseq** [comment]
@tchaton Dear Thomas, 
 thank you for your interest and kind words! 

We always appreciate any integrations, they move our software into the right direction :slightly_smiling_face:
Have you explored our ML integrations already? If not, you can check this docs: 
https://labelstud.io/guide/ml.html & https://labelstud.io/guide/ml_create.html

I think we can develop a tutorial about Flash and put it here in case of success: 
https://labelstud.io/guide/ml_tutorials.html

Cheers, 
  Max.
